### PR TITLE
HLA-837: Invalid Reference to Pyraf

### PR DESCRIPTION
**Pull request by s-goldman**
Resolves [HLA-837](https://jira.stsci.edu/browse/HLA-837)

This PR changes outdated mentions of pyraf to python. It also remove several outdated comments. 

This PR also adds a couple log files to the .gitignore file that tend to be accidentally added to commits. 

**Checklist for maintainers**
- [x] added entry in `CHANGELOG.rst` within the relevant release section
- [x] updated or added relevant tests
- [x] updated relevant documentation
- [x] added relevant milestone
- [x] added relevant label(s)
- [ ] ran regression tests, post a link to the Jenkins job below.
      [How to run regression tests on a PR](https://github.com/spacetelescope/jwst/wiki/Running-Regression-Tests-Against-PR-Branches)

### CODE REVIEW

File: drizzlepac/astrodrizzle.help [hunk 1]
```diff
@@ -1009,33 +1008,28 @@ ignore pixel flags of 64 and 32 in the DQ array of the (``_flt.fits``)
...
     ...     final_scale=0.05, final_rot=0)
 
-   Or, run the same task from the PyRAF command line, but specify all
+   Or, run the same task from the python command line, but specify all
    parameters in a config file named ``myparam.cfg``:
 
     >>> astrodrizzle.AstroDrizzle('*flt.fits', configobj='myparam.cfg')
 
-3. Run the task directly from Python:
+2. Run the task directly from Python:
```
Comment (mdlpstsci) [inline drizzlepac/astrodrizzle.help:1024]: Item 2 is the same as lines 1013-1017 from Item 1. Please remove the redundancy.

File: drizzlepac/astrodrizzle.help [hunk 2]
```diff
@@ -1009,33 +1008,28 @@ ignore pixel flags of 64 and 32 in the DQ array of the (``_flt.fits``)
...
 
     >>> import drizzlepac
     >>> from drizzlepac import astrodrizzle
     >>> astrodrizzle.AstroDrizzle('*flt.fits', output='final',
     ...     wcskey='A', driz_sep_bits='64,32', final_wcs=True,
     ...     final_scale=0.05, final_rot=0)
 
-   Or, run the same task from the PyRAF command line, but specify all
+   Or, run the same task from the python command line, but specify all
    parameters in a config file named ``myparam.cfg``:
```
Comment (mdlpstsci) [inline drizzlepac/astrodrizzle.help:1020]: I just want to make sure you tested the use of the configuration file in this context.  Please comment.

File: drizzlepac/astrodrizzle.help [hunk 3]
```diff
@@ -1009,33 +1008,28 @@ ignore pixel flags of 64 and 32 in the DQ array of the (``_flt.fits``)
...
 
     >>> from drizzlepac import astrodrizzle
     >>> astrodrizzle.AstroDrizzle('*flt.fits', output='final',
     ...     wcskey='A', driz_sep_bits='64,32', final_wcs=True,
     ...     final_scale=0.05, final_rot=0)
 
-4. Help can be accessed via the "Help" pulldown menu in the ``TEAL`` GUI.
-   It can also be accessed from the ``PyRAF`` command-line and saved
+3. Help can be accessed via the "Help" pulldown menu in the ``TEAL`` GUI.
+   It can also be accessed from the python command-line and saved
```
Comment (mdlpstsci) [inline drizzlepac/astrodrizzle.help:1032]: Did you check this help invocation still works correctly?  The "page" command does not work for me in the basic Python shell.

File: drizzlepac/pixtosky.help [hunk 4]
```diff
@@ -75,31 +75,24 @@ stwcs
...
-1. Run the task using the TEAL GUI under PyRAF:
 
-   >>> import drizzlepac
-   >>> epar pixtosky
-
-2. Convert a single X,Y position (100,100) from an calibrated ACS image
-   (``j8bt06nyq_flt.fits``) into an undistorted sky position (RA,Dec)
-   without using the TEAL GUI:
+1. Convert a single X,Y position (100,100) from an calibrated ACS image
+   (``j8bt06nyq_flt.fits``) into an undistorted sky position (RA,Dec)å:
```
Comment (mdlpstsci) [inline drizzlepac/pixtosky.help:83]: Delete character at the end of line 83.

File: drizzlepac/tweakreg.help [hunk 5]
```diff
@@ -647,17 +642,12 @@ Save the residual offsets (``dx``, ``dy``, ``rot``, ``scale``, ``xfit_rms``, ``y
 
     >>> tweakreg.edit_imagefindpars()
 
-3. Help can be accessed via the "Help" pulldown menu in the TEAL GUI.  It can also
-   be accessed from the PyRAF command-line and saved to a text file:
+2. Help can be accessed via the "Help" pulldown menu in the TEAL GUI.  It can also
+   be accessed from the python command-line and saved to a text file:
```
Comment (mdlpstsci) [inline drizzlepac/tweakreg.help:646]: The text has been removed which actually shows the help information being written to a file.

File: drizzlepac/astrodrizzle.help (hunk 2 above)
Comment (s-goldman) [reply drizzlepac/astrodrizzle.help:1020]: I did not originally, but I did this morning. 

The code runs and attempts to load the custom configuration file using `teal`, but throws an error when run `teal.load()`. 

That being said, i've never been able to run teal on my Mac.

File: drizzlepac/astrodrizzle.help (hunk 3 above)
Comment (s-goldman) [reply drizzlepac/astrodrizzle.help:1032]: It doesn't for me either. That line has been removed.

**Review by mdlpstsci — APPROVED** [review]
Many good updates and clean-up of cruft!  Once most of the dust has settled, we may have to sweep through the docs again, but it will be soooooo much easier!

**Pull request merged**